tests/test_tatsu.py

```diff
@@ -0,0 +1,13 @@
+from jinja2 import Environment
```

Comment:
Cool, this is a really good start for testing.

For the next step, I'd like to see many of the existing lexer's tests duplicated.

In order to do that, I think it will be good to use a customized test case (instead of just copying them here). I'll send you links separately via Discord.